dataactbroker/helpers/validation_helper.py

```diff
@@ -265,6 +267,27 @@ def expected_length(row, csv_schema):
     return 'Max length: {}'.format(current_field.length)
 
 
+def valid_format(row):
```

Comment:
This is pretty confined to just date formatting. For future reference, if there are other strings/types to format, is the plan to add the new type to the `parsed_fields['format']` and then in this function chooses the appropriate format for the given type? Otherwise, if this is just going to be used for `date`s, then I'd ask to update the name to `valid_date_format`.